src/onnx/onnx.cpp

```diff
@@ -1816,9 +1826,9 @@ struct onnx_parser
             }
             else
             {
-                assert(node.output().size() <= result.size());
+                auto output_num = std::min<std::size_t>(node.output().size(), result.size());
```

Comment:
Why was this changed?

Reply:
the rnn operator satisfies node.output().size() > result.size(), but the MaxPooling operator tis the case node.output().size() > result.size().